src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx

```diff
@@ -32,27 +33,51 @@ export function ReceiveCollectibleOrdinal() {
           <Title fontSize="20px" lineHeight="28px" px="extra-loose">
             Deposit Ordinal inscription
           </Title>
-          <ErrorLabel>Important to keep in mind</ErrorLabel>
-          <Text color={color('text-title')} fontSize={1} px={['base', '40px']}>
-            This Taproot address is for Ordinals only. Do not use this address more than once for
-            depositing an inscription
+          <Text color={color('text-title')} fontSize={1} px={['50px', '70px']}>
+            Use your wallet to create inscriptions within minutes in
+          </Text>
+          <Stack alignItems="center" pb={['unset', 'base']}>
+            <Stack
+              _hover={{ cursor: 'pointer' }}
+              alignItems="center"
+              as="button"
+              color={color('accent')}
+              fontSize={1}
+              isInline
+              onClick={() => openInNewTab('https://gamma.io/')}
+              spacing="extra-tight"
+              textDecoration="underline"
+              type="button"
+            >
+              <Text color="inherit">Gamma.io</Text>
+              <Box as={FiArrowUpRight} />
+            </Stack>
+            <Stack
+              _hover={{ cursor: 'pointer' }}
+              alignItems="center"
+              as="button"
+              color={color('accent')}
+              fontSize={1}
+              isInline
+              onClick={() => openInNewTab('https://ordinalsbot.com/')}
+              spacing="extra-tight"
+              textDecoration="underline"
+              type="button"
+            >
+              <Text color="inherit">ordinalsbot.com</Text>
+              <Box as={FiArrowUpRight} />
+            </Stack>
+          </Stack>
+          <Divider />
+          <ErrorLabel mt={['unset', 'base']}>Important to keep in mind</ErrorLabel>
+          <Text color={color('text-title')} fontSize={1} px={['unset', 'loose']}>
+            This Taproot address is for Ordinals only. Do not deposit other BTC here or you may have
+            difficulty retrieving it
           </Text>
         </Stack>
-        <Flex alignItems="center" justifyContent="center" my="extra-loose" width="100%">
-          <Button
-            borderRadius="10px"
-            height="48px"
-            mode="tertiary"
-            mr="base"
-            onClick={() => navigate(RouteUrls.ReceiveCollectible)}
-            type="button"
-          >
-            Go back
-          </Button>
-          <PrimaryButton flexGrow={1} onClick={copyToClipboard}>
-            Proceed to copy address
-          </PrimaryButton>
-        </Flex>
+        <PrimaryButton flexGrow={1} my="extra-loose" onClick={copyToClipboard} width="100%">
+          Proceed to copy address
```

Comment:
Imo this is too verbose, shouldn't it just be Copy address? @markmhx

Reply:
yep shorter works for me